release_notes/unreleased.md

```diff
@@ -1 +1,3 @@
 **Unreleased**
+* Added new optional parameter 'credential id' for scanning with a credential [PAPP-27075]
```

Comment:
Suggestion :   Added new optional parameter 'credential id' for scanning with a credential [PAPP-27075]